commands/sync.go

```diff
@@ -0,0 +1,62 @@
+package commands
+
+import (
+	"github.com/bndw/pick/backends"
+	"github.com/bndw/pick/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "sync [other-pick-safe]",
+		Short: "Sync current safe with another pick safe",
+		Long:  "The sync command is used to sync the current pick safe with another pick safe.",
+		Run: func(cmd *cobra.Command, args []string) {
+			runCommand(Sync, cmd, args)
+		},
+	})
+}
+
+func Sync(args []string, flags *pflag.FlagSet) error {
+	otherSafePath, err := parseSyncArgs(args)
+	if err != nil {
+		return err
+	}
+
+	safeLoader := newSafeLoader()
+
+	safe, err := safeLoader.LoadSafe()
```

Comment:
`safeLoader.LoadSafe()` seems redundant. Could the method just be called `safeLoader.Load`?